sw/device/tests/alert_handler_lpg_clkoff_test.c

```diff
@@ -484,8 +495,8 @@ bool test_main(void) {
       test_phase = test_step_cnt / ARRAYSIZE(kPeripherals);
       peri_idx = test_step_cnt - (test_phase)*ARRAYSIZE(kPeripherals);
 
-      // Wait for a random time <= 1024us
-      busy_spin_micros(rand_testutils_gen32_range(1, 1024));
+      // Wait for a random time <= 10us
```

Comment:
@cindy just reduced the waiting time. My original intention was to wait up to 1000 clock cycles just in case.